.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -182,7 +182,7 @@ jobs:
   aws-start:
     name: Start AWS
     needs: build
-    if: ${{ inputs.lts_aws_matrix != '[]' }}
+    if: ${{ always() && needs.build.result == 'success' && inputs.lts_aws_matrix != '[]' }}
```

Comment:
"That resulted in the aws-start being skipped"
Will this change run only `aws-start` or E2E tests for all backends?

Reply:
My intention is for the latter. Need to wait for the testing to finish to verify. Note that `e2e-tests` already has the same `if:` guard at line 208.

Reply:
Note that we skip aws in post-commit because `lts_aws_matrix` is empty there (we only do testing on Intel HW in post-commit).